frontend/views/containers/chatroom/image-viewer/PreviewImageArea.vue

```diff
@@ -5,7 +5,7 @@
   @mouseup='mouseUpHandler'
 )
   img.c-preview-image(ref='previewImg'
-    :class='{ "is-movable": isImageMovable }'
+    :class='{ "is-movable": isImageMovable, "is-hidden": !ephemeral.imgInitDone }'
```

Comment:
Discovered there was a bug where the image kind of flickers when it finishes the load/initialization steps. So made a fix here.

Reply:
Awesome! I noticed this behavior too but thought it was Android's fault.